ModernWallet/Classes/ViewController/CashOut/CashOutEnterAmountViewController.swift

```diff
@@ -71,6 +71,24 @@ class CashOutEnterAmountViewController: UIViewController {
             .drive(assetImageView.rx.afImage)
             .disposed(by: disposeBag)
         
+        let isAssetPairExistDriver = walletViewModel.isAssetPairExist.asDriver(onErrorJustReturn: false)
```

Comment:
Suggestion: move the conversion to `Driver` in the view model.
Also, please rename the variables - `isAssetPairNull` sounds a bit better, and you can omit the 'driver' part in the name.
Example: `let isAssetPairNull = walletViewModel.isAssetPairNull`

Extra tip: if you consider my suggestions, you can drive the view model property directly, without assigning it to a local constant.